content/guides/embed/ui-elements/explorer.md

```diff
@@ -264,7 +271,175 @@ more, see [Dedicated Scopes for Box UI Elements][scopes].
 | User want basic functionality + preview + download + rename file/folder names             | `base_explorer` + `item_preview` + `item_download` + `item_rename`                                                |
 | User wants all functionality (basic, preview, download, rename, share, upload and delete) | `base_explorer` + `item_preview` + `item_download` + `item_rename` + `item_delete` + `item_share` + `item_upload` |
 
+## Metadata view
+
+With Content Explorer you can also display files
+and folders based on their metadata.
+This view is called the metadata view and uses
+metadata template and metadata query to find the data you
+want to display.
+
+### Prerequisites
+
+- Node version 18.18.2 or higher
```

Comment:
Could you check with Greg the current supported Node version by BUIE, I think we are not supporting 20 yet. And the demo worked on 18.16.0. I think that would be 18.0.0 or higher, but not 20.

Reply:
it's 19, just released :)